game/factions/us_aggressors.py

```diff
@@ -0,0 +1,74 @@
+from dcs.helicopters import *
+from dcs.planes import *
+from dcs.ships import *
+from dcs.vehicles import *
+
+US_Aggressors = {
+    "country": "USA",
```

Comment:
Country should be "USAF Aggresors", see in pydcs the name of the faction : https://github.com/pydcs/dcs/blob/a5ace97ed1b13f56f8ec20e569a30898ec5f2e3b/dcs/countries.py#L1906

Otherwise, if you try to generate a mission with for instance "USA 2005" vs "US Aggresors", I believe the mission file is going to be corrupted because both coalition will contain the same "USA" faction. So for now, it's important to make sure the factions on the red side do not use the same "countries" as the blue ones. (But i do want to change that so you can operate any faction from any side.)